worlds/timespinner/Locations.py

```diff
@@ -18,257 +18,257 @@ def get_locations(world: Optional[MultiWorld], player: Optional[int]) -> Tuple[L
+        LocationData('Caves of Banishment (upper)', 'Caves of Banishment: Pedestal',  1337120),
+        LocationData('Caves of Banishment (Maw)', 'Caves of Banishment: Last chance before Maw',  1337121, lambda state: state._timespinner_has_doublejump(world, player)),
+        LocationData('Caves of Banishment (Maw)', 'Caves of Banishment: Plasma Crystal', 1337173, lambda state: state.has_any({'Gas Mask', 'Talaria Attachment'}, player)),
         LocationData('Caves of Banishment (Maw)', 'Killed Maw',  EventId, lambda state: state.has('Gas Mask', player)),
-        LocationData('Caves of Banishment (Maw)', 'Mineshaft',  1337122, lambda state: state.has('Gas Mask', player)),
-        LocationData('Caves of Banishment (Sirens)', 'Wyvern room',  1337123),
-        LocationData('Caves of Banishment (Sirens)', 'Upper banishment above sirens',  1337124),
-        LocationData('Caves of Banishment (Sirens)', 'Under banishment sirens left',  1337125, lambda state: state.has('Water Mask', player)),
-        LocationData('Caves of Banishment (Sirens)', 'Under banishment sirens right',  1337126, lambda state: state.has('Water Mask', player)),
-        LocationData('Caves of Banishment (Sirens)', 'Underwater banishment sirens right ground', 1337172, lambda state: state.has('Water Mask', player)),
-        LocationData('Caves of Banishment (Sirens)', 'Banishment water hook',  1337127, lambda state: state.has('Water Mask', player)),
-        LocationData('Castle Ramparts', 'Castle bomber chest',  1337128, lambda state: state._timespinner_has_multiple_small_jumps_of_npc(world, player)),
-        LocationData('Castle Ramparts', 'Ramparts Freeze the engineer',  1337129, lambda state: state.has('Talaria Attachment', player) or state._timespinner_has_timestop(world, player)),
-        LocationData('Castle Ramparts', 'Ramparts Giantess guarded room',  1337130),
-        LocationData('Castle Ramparts', 'Ramparts Knight and archer guarded room',  1337131),
-        LocationData('Castle Ramparts', 'Ramparts pedestal',  1337132),
-        LocationData('Castle Keep', 'Castle basement secret pedestal',  1337133, lambda state: state._timespinner_can_break_walls(world, player)),
-        LocationData('Castle Keep', 'Clean the castle basement',  1337134),
-        LocationData('Royal towers (lower)', 'Yas queen room',  1337135, lambda state: state._timespinner_has_pink(world, player)),
-        LocationData('Castle Keep', 'Castle basement giantess',  1337136),
-        LocationData('Castle Keep', 'Omelette chest',  1337137),
-        LocationData('Castle Keep', 'Just an egg',  1337138),
-        LocationData('Castle Keep', 'Under the twins',  1337139),
+        LocationData('Caves of Banishment (Maw)', 'Caves of Banishment: Mineshaft',  1337122, lambda state: state.has('Gas Mask', player)),
+        LocationData('Caves of Banishment (Sirens)', 'Caves of Banishment: Upper wyvern room',  1337123),
```

Comment:
if anything just call this "Caves of Banishment Sirens", not "Caves of Banishment: Upper sirens"
same for the other siren locations

Reply:
My reasoning was there are technically rooms in the bottom section of Banishment that have sirens and wyverns. Yes there are no checks in those ones but this helps newer players in finding the right room, as well as keeps it close to what the original check name was. Im not married to the idea though